### DISCUSSION TITLE
Welcome to PicoGK Discussions!

### DISCUSSION THREAD

**Discussion by leap71** [leap71/PicoGK#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Intaekjo** [comment]
Hi Leap71 team,
I tried to setup working environment. Looks like windows dll is missing in zip folder although manual described in below. Do you have any plan for windows?

Now, let's install the PicoGK library.

Download [the ZIP from this link](https://github.com/leap71/PicoGK/archive/refs/heads/main.zip), which is the PicoGK library.
Copy the ZIP to the folder which contains Program.cs (in the project you just created)
Unzip it. Now your folder should contain Program.cs, and a subfolder named PicoGK
Go into the PicoGK folder, and navigate to the subfolder Runtime
If you are on Mac, copy the .dylib files you see there to /usr/local/lib
If you are on Windows, copy the .dll files to C:\SYSTEM32

**Reply by LinKayser** [reply]
You can now use the installers available at: https://github.com/leap71/PicoGK/releases/tag/PicoGK-v1.1.0

**Comment by LinKayser** [comment]
Dear @Intaekjo — we have not provided pre-compiled DLLs for Windows yet. You can, however, compile PicoGKRuntime yourself, if you like — or wait a few weeks until we get to it.

Best,
Lin
